tabs/osd.js

```diff
@@ -591,12 +653,22 @@ OSD.msp = {
     }
     return result;
   },
-  encode: function(display_item) {
+  encodeLayout: function(display_item) {
     var buffer = [];
+    if (semver.gte(CONFIG.apiVersion, "1.36.0")) {
```

Comment:
This should really go to the end of the message, to keep compatibility with the existing format. See also comments on https://github.com/betaflight/betaflight/pull/3133